### PR TITLE
added validation in inflate + test cases (need more)

### PR DESCRIPTION
**Pull request by jonathankingfc**

### CODE REVIEW

File: pkg/kustomize/kustomize_test.go [hunk 1]
```diff
@@ -194,16 +195,75 @@ var inflateTests = []struct {
...
 func TestInflate(t *testing.T) {
-	assert := assert.New(t)
 
 	for _, test := range inflateTests {
 		pieces, err := Inflate(test.quayRegistry, test.configBundle)
 
-		assert.NotNil(pieces)
-		assert.Equal(len(test.expected), len(pieces))
-		assert.Nil(err)
+		// Inflate returned null object
```
Comment (alecmerdler) [inline pkg/kustomize/kustomize_test.go:239]: Unnecessary comment.

File: pkg/kustomize/kustomize_test.go [hunk 2]
```diff
@@ -194,16 +195,75 @@ var inflateTests = []struct {
...
+		if pieces == nil {
+			t.Errorf("Test %s failed. Inflate returned a null object", test.name)
+			return
+		}
+
+		// Did not expect error
```
Comment (alecmerdler) [inline pkg/kustomize/kustomize_test.go:245]: Unnecessary comment.

File: pkg/kustomize/kustomize_test.go [hunk 3]
```diff
@@ -194,16 +195,75 @@ var inflateTests = []struct {
...
+		if err != nil && test.expectedErr == nil {
+			t.Errorf("Test %s failed. Expected no error but received error: %s", test.name, err.Error())
+			return
+		}
+
+		// Expected error
```
Comment (alecmerdler) [inline pkg/kustomize/kustomize_test.go:251]: Unnecessary comment.

File: pkg/kustomize/kustomize_test.go [hunk 4]
```diff
@@ -194,16 +195,75 @@ var inflateTests = []struct {
...
+		if err != nil && test.expectedErr != nil {
+
```
Comment (alecmerdler) [inline pkg/kustomize/kustomize_test.go:253]: No space before function body.

File: pkg/kustomize/kustomize_test.go [hunk 5]
```diff
@@ -194,16 +195,75 @@ var inflateTests = []struct {
...
+			// Wrong error
```
Comment (alecmerdler) [inline pkg/kustomize/kustomize_test.go:254]: Unnecessary comment.

File: pkg/kustomize/kustomize_test.go [hunk 6]
```diff
@@ -194,16 +195,75 @@ var inflateTests = []struct {
...
+			if err.Error() != test.expectedErr.Error() {
+				t.Errorf("Test %s failed. Expected error: %s. Received error: %s", test.name, test.expectedErr.Error(), err.Error())
+			}
+
+			// Correct error
```
Comment (alecmerdler) [inline pkg/kustomize/kustomize_test.go:259]: Unnecessary comment.

File: pkg/kustomize/kustomize_test.go [hunk 7]
```diff
@@ -194,16 +195,75 @@ var inflateTests = []struct {
...
+			return
+		}
+
+		// Make sure inflate returns correct number
```
Comment (alecmerdler) [inline pkg/kustomize/kustomize_test.go:263]: Unnecessary comment.

**Comment by jonathankingfc** [comment]
Opening new PR

**Pull request closed without merging**